.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -40,12 +40,12 @@ body:
       label: Expected Behavior
       description: |
         What did you expect to happen? For example:
-        - I expected that the PowerShell script wouldn't give me this error message.
+        - I expected the PowerShell script wouldn't give me this error message.
```

I think that we need a preposition here for this to be grammatical.
```suggestion
        - I expected that the PowerShell script wouldn't give me this error message.
```